### PR TITLE
Added file existence checking before adding to sprite.

### PR DESCRIPTION
**Pull request by Kreeg**
Found here #759
@XhmikosR I think those code scanning was correct. All the other file openings in bin file are wrapped in try...catch. So it's good to add another protection

### CODE REVIEW

**Comment by Kreeg** [comment]
@XhmikosR sorry that was too early. I'll wrap it in try..catch instead

File: bin/svg-sprite.js [hunk 1]
```diff
@@ -276,7 +276,17 @@ argv._.reduce((f, g) => [...f, ...glob.sync(g)], [])
             basename = basepos >= 0 ? basename.substr(basepos + 2) : path.basename(file);
         }
 
-        spriter.add(file, basename, fs.readFileSync(file));
+        let openedFile = null;
+
+        try {
+            openedFile = fs.readFileSync(file);
+        } catch (error) {
+            console.error(`[ERROR] Skipping (${file}) due to errors (${error.message.trim()})`);
```
Comment (XhmikosR) [inline bin/svg-sprite.js:284]: Wouldn't it be better to throw here?

It's hard working without tests for bin, but I assume you tested it anyway :)

Comment (Kreeg) [reply bin/svg-sprite.js:284]: I think you're right. It's better to throw. But github suggest using file descriptors instead. 
I think we can dismiss this alert

Comment (XhmikosR) [reply bin/svg-sprite.js:284]: Sounds good to me. We can revisit it later if needed.

**Pull request closed without merging**